dataminer-overview/Pricing/Pricing_Perpetual_Use_Licensing.md

```diff
@@ -25,7 +25,8 @@ Depending on the application, licensing is available as **server-based licenses*
 | Management of high-volume **service-delivery endpoints**.<br>From 25,000 up to several millions of endpoints. | Experience and Performance Management, Volume-based | 25K, 50K, 100K, 250K, 500K, 1000K service-delivery endpoints. |
 | Local Management of **remote assets**.<br>From 5 to 50 objects per remote location. | DataMiner Probe, Server-based | 5, 10, 25, 50 Managed Objects |
 
-> [!IMPORTANT]
+
+> [!WARNING]
```

Comment:
Just to be sure: should this definitely be a warning? Warnings are usually only used for actions with dangerous consequences, that could for instance lead to damage to the system. If it should indeed be a warning, it might be better to expand the warning with more information about the relevant risks.